testUtils/mongoMocker.js

```diff
@@ -0,0 +1,95 @@
+import EventEmitter from 'events';
+import mongodb from 'mongo-mock';
+import { hook } from '../src/extender';
+
+export const MongoMockerEventEmitter = (() => {
+  let eventEmitter = new EventEmitter();
+  const getEventEmitter = () => eventEmitter;
+  const cleanEventEmitter = () => {
+    eventEmitter.eventNames().forEach(event => {
+      eventEmitter.removeAllListeners(event);
+    });
+  };
+  return { getEventEmitter, cleanEventEmitter };
+})();
+
+export const wrapMongoCollection = (collection, funcName, failed = false) => {
+  hook(collection, funcName, {
+    beforeHook: args => {
+      MongoMockerEventEmitter.getEventEmitter().emit('started', {
+        eventName: 'onStartedHook',
+        command: {
+          insert: 'documents',
+          documents: args[0],
+          ordered: true,
+          lsid: { id: '2' },
+          txnNumber: 1,
+          $clusterTime: { clusterTime: 123, signature: { '1': 1 } },
+          $db: 'TracerDB',
+        },
+        databaseName: 'TracerDB',
+        commandName: funcName,
+        requestId: 13,
+        operationId: undefined,
+        connectionId: 1,
+      });
+    },
+    afterHook: () => {
+      if (failed) {
+        MongoMockerEventEmitter.getEventEmitter().emit('failed', {
+          duration: 26,
+          failure: 'Wow What a error',
+          commandName: funcName,
+          requestId: 13,
+          operationId: undefined,
+          connectionId: 1,
```

Comment:
All these variables are a mocked response from MongoDB, right?
Is there a way to validate that the mock is really what Mongo will return? Maybe something with `mongo-mock`?